.changeset/pretty-ligers-bathe.md

```diff
@@ -0,0 +1,11 @@
+---
+'@swisspost/design-system-components-angular': minor
```

Comment:
This will release as 1.0.0 (see change request in package.json), which gives a slightly better changelog, starting with v1.0.0 instead of v1.1.0.

```suggestion
'@swisspost/design-system-components-angular': major
```